docs/sdks/server-sdks/node.md

```diff
@@ -330,30 +330,207 @@ To read more about different flag types, see the page on [Flag Variations](/feat
-| `holdoutVariation` (string)     | An Eppo holdout variation if experiment is eligible for analysis key                                                                                                   | "status_quo", "all_shipped_variations", or null                    |
+| Field                            | Description                                                                                                              | Example                                         |
+|----------------------------------|--------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------|
+| `experiment` (string)            | An Eppo experiment key                                                                                                   | "recommendation-algo-allocation-17"             |
+| `subject` (string)               | An identifier of the subject or user assigned to the experiment variation                                                | UUID                                            |
+| `variation` (string)             | The experiment variation the subject was assigned to                                                                     | "control"                                       |
+| `timestamp` (string)             | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z                        |
+| `subjectAttributes` (Attributes) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`                           |
+| `featureFlag` (string)           | An Eppo feature flag key                                                                                                 | "recommendation-algo"                           |
+| `allocation` (string)            | An Eppo allocation key                                                                                                   | "allocation-17"                                 |
+| `holdout` (string)               | An Eppo holdout group key                                                                                                | "q1-holdout"                                    |
+| `holdoutVariation` (string)      | An Eppo holdout variation if experiment is eligible for analysis key                                                     | "status_quo", "all_shipped_variations", or null |
+
+:::note
+The `Attributes` type represents a mapping of an attribute name to its value, which could be a string, number or boolean (`Record<string, string | number | boolean>`).
+:::
 
 :::note
 More details about logging and examples (with Segment, Rudderstack, mParticle, Snowplow, Amplitude) can be found in the [event logging](/sdks/event-logging/) page.
 :::
+
```

Comment:
👇 New bandit content starts here 👇 

(May be easiest to review using the auto-generated preview link auto-commented in the PR discussion)

Reply:
2 comments:

1. It feels like the concept of 'actions' is still vague to me. I can anticipate customers asking what an action does and what outcomes actions affect in the bandit. A couple suggestions:
  a. Include an explanation of what is done in the Eppo UI and what is required or can be done with the SDK. For example, the string for the variation name comes from the Eppo UI and the bandit actions are included with the SDK
  b. Could we add an example output for the getBanditAction() method similar to what exists for the Node feature flag example?
2. Java docs note that actions are [optional](https://deploy-preview-418--eppo-data-docs.netlify.app/sdks/server-sdks/java/#pass-bandit-actions-to-request-a-bandit-assignment), but I don't see that stated in the Node docs.

Other than that, looks great!

Reply:
Thanks for the comments!

Regarding 1, happy to be more clear about where things are specified as you're correct that is confusing. Variations, allocations, and targeting rules are all set up in the UI. Same with the bandit's key. However actions are provided via the SDK only. 

Regarding 2, that is because the Java approach folded bandits into the `getStringAssignment()` function. With python and other UFC SDKs, we decided to add the "syntactic sugar" of an explicit `getBanditAction()` method, and then having that be the only entry point (for now) for querying bandits.

Reply:
Ahh ha I see re: 2, good to know. 

other than that looks good to ship for me!